src/Http/Client/Response.php

```diff
@@ -236,6 +236,16 @@ public function isOk()
         return $this->code >= 200 && $this->code <= 399;
     }
 
+    /**
+     * Check if the response status code was in the 2xx range
+     *
+     * @return bool
+     */
+    public function isSuccessful()
```

Comment:
Nitpick: Can we have a shorter name `isSuccess()`? :slightly_smiling_face:

Reply:
Sure